File: Célia_LingProgram/src/LISTA02/Exer01_Pizzada.java
package LISTA02;

import java.util.Scanner;

public class Exer01_Pizzada {
    public static void main(String[] args) {

        Double preco, max, APagar;
        String sabor;
        Integer pessoas;

        Scanner leitor = new Scanner(System.in);
        Scanner leitorS = new Scanner(System.in);


        System.out.println("CalculadoraPizza");
        System.out.println("Insira o valor de uma pizza:");
        preco = leitor.nextDouble();
        System.out.println("Insira o sabor da pizza:");
        sabor = leitorS.next();
        System.out.println("Insira a quantidade de pessoas que vão dividir " +
                "o valor da pizza:");
        pessoas = leitor.nextInt();
        System.out.println("Insira o valor maximo que cada um pode pagar" +
                " pela pizza:");
        max = leitor.nextDouble();

        APagar = preco / pessoas;

        if (APagar <= max) {
            System.out.println("A pizza de " + sabor + "será divida entre " +
                    pessoas + " pessoa(s). R$" + APagar + " para cada.");
        }else{
            System.out.println("Deu ruim no racha. A pizza deveria custar no" +
                    " máximo "+max+ " para cada um.");
        }
    }
}

File: Célia_LingProgram/ListaJavaFacul/src/L2_Exer01_Pizzada/Exer01_Pizzada.kt
package Lista02

import java.util.*

//Crie uma classe chamada “CalculadoraPizza” que:
//A. Solicite o valor de uma pizza.
//B. Solicite o sabor da pizza.
//C. Solicite a quantidade de pessoas que irá "rachar" a conta.
//D. Solicite o valor máximo que as pessoas aceitam "rachar" .
//E. Caso o valor da pizza seja possível de "rachar", Exibir:
//“A pizza de SABOR será dividida entre X pessoas. R$VALOR p/ cada"
//F. Caso o "racha" seja impossível exibir:
//"Deu ruim no racha. A pizza deveria custar no máximo X"
//Onde X é o valor máximo que uma pizza pode custar.

fun main() {

    val reader = Scanner(System.`in`)
    val readerString = Scanner(System.`in`)

    println("CalCuladoraPizza")
    println("Insira o valor da pizza: ")
    val valorPizza: Double = reader.nextDouble()
    println("Insira o sabor da pizza: ")
    val saborPizza: String = readerString.nextLine()
    println("Insira quantas pessoas irão rachar: ")
    val quantPessoas: Int = reader.nextInt()
    println("Insira o valor máximo que cada um aceita pagar: ")
    val valorRacha: Double = reader.nextDouble()

    val valorPagar: Double = valorPizza / quantPessoas

    if (valorPagar > valorRacha) {
        println("Deu ruim no racha. A pizza deveria custar no máximo " + valorRacha)
    } else {
        println("A pizza de " + saborPizza + " será dividida entre " + quantPessoas + " pessoas. R$" + valorPagar + " p/ cada")
    }
}